packages/flutter/lib/src/material/switch.dart

```diff
@@ -46,14 +46,17 @@ class Switch extends StatefulWidget {
   ///
   /// * [value] determines whether this switch is on or off.
   /// * [onChanged] is called when the user toggles the switch on or off.
-  const Switch({
-    Key key,
-    @required this.value,
-    @required this.onChanged,
-    this.activeColor,
-    this.activeThumbImage,
-    this.inactiveThumbImage
-  }) : super(key: key);
+  const Switch(
+      {Key key,
+      @required this.value,
+      @required this.onChanged,
+      this.activeColor,
+      this.activeTrackColor,
+      this.inactiveThumbColor,
+      this.inactiveTrackColor,
+      this.activeThumbImage,
+      this.inactiveThumbImage})
+      : super(key: key);
```

Comment:
nit: please use the original style, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo for details on our style